src/assessment.handlebars

```diff
@@ -210,7 +210,7 @@
   </div>
   <div class="slide are-you-sure" id="are-you-sure">
     <div class="box">
-      <h1 tabindex="-1" style="red">Are you sure?</h1>
```

Comment:
lol

Reply:
@katjam couldn't deal with it after you pointed it out 😅